### DISCUSSION TITLE
Interest in an Artery workshop at OMNeT++ Community Summit?

### DISCUSSION THREAD

**Discussion by riebl** [riebl/artery#209] [General]
Hi all,

Are you interested in having a workshop on Artery at this year's [OMNeT++ Community Summit](https://summit.omnetpp.org/2021/index.html)?
If a small group of interested people exists, I would try to organise a workshop, e.g. as a part of the hackathon event.
I am looking forward to your feedback!

Raphael

**Comment by xrispa-m** [comment]
Definitely!! It would be really interesting to have a workshop dedicated on Artery!!

**Comment by kenog** [comment]
Hi Raphael,

that sounds like a nice idea. I'd be interested :)

Best regards
Keno

**Comment by awillecke** [comment]
I would be interested as well. 

Do you already have a topic in mind for a hackathon?

Regards
Alex

**Reply by riebl** [reply]
No, I have nothing specific in mind yet. Any suggestions are welcome.

**Reply by awillecke** [reply]
I haven't really looked into `libsumo` yet, but as you mentioned in #213, Artery should switch soonish.
Do you think this would be a feasible topic to work on collaboratively? Depending on the participants, it also might be a bit stale...

**Reply by riebl** [reply]
That could be a topic. However, I have not yet wrapped my mind around how to realize such a feature during a hackathon. Depending on the participants, we can just pick any open "enhancement" ticket. So feel free to add more such tickets :-)

**Comment by riebl** [comment]
The summit program is now available at https://summit.omnetpp.org/2021/index.html#program.
We have two slots from 13:00 to 17:30 (CEST) on 9th September. Do not forget to register at the summit website to attend our hackathon!
See you then!
